### ISSUE TITLE
Supported file formats not specified in README

### ISSUE THREAD

**Issue by eschoenawa** [potree/PotreeConverter#592]
The README file of the project doesn't specify the supported file formats. To me, it seems like that is important information for this kind of software.

**Comment by Cornelius000** [comment]
I agree, i am searching as well right now. Does anyone know what files formats are supported?

**Comment by mf1989** [comment]
I have the same issue as @eschoenawa and @Cornelius000 .
